Makefile

```diff
@@ -27,104 +27,71 @@ RUBY_ENV += RUBY_GC_OLDMALLOC_LIMIT_GROWTH_FACTOR=1.2
 
 default: all
 
-COMPOSE_PROJECT_NAME := $(PROJECT)
-COMPOSE_FILE := docker-compose.yml
-COMPOSE_TEST_FILE := docker-compose.test-$(DB).yml
-
 ## This image is private and cannot be accessed by another third party than redhat.com employees
 ## You will need to build your own image as instructed in https://github.com/oracle/docker-images/tree/master/OracleDatabase/SingleInstance
 ORACLE_DB_IMAGE := quay.io/3scale/oracle:12.2.0.1-ee
 
 include wget.mk
 include openshift.mk
 
-.PHONY: default all clean build test info docker test-run tmp-export run test-bash clean-cache clean-tmp compose help
+.PHONY: assets bash build bundle clean dev-setup dev-start dev-stop help oracle-database oracle-db-setup run schema yarn
 .DEFAULT_GOAL := help
 
 # From here on, only phony targets to manage docker compose
-all: clean-tmp dev-setup dev-start
-
-info: docker # Prints relevant environment info
+all: dev-setup dev-start
 
-docker: ## Prints docker version and info
-	@echo
-	@echo "======= Docker ======="
-	@echo
-	@docker version
-	@echo
-	@docker info
-	@echo
-
-test-run: ## Runs test inside container
-test-run: COMPOSE_FILE = $(COMPOSE_TEST_FILE)
-test-run: $(DOCKER_COMPOSE) clean-tmp cache
-	$(DOCKER_COMPOSE) run --name $(PROJECT)-build $(DOCKER_ENV) build $(CMD)
-
-tmp-export: ## Copies files from inside docker container to local tmp folder.
-tmp-export: IMAGE ?= $(PROJECT)-build
-tmp-export: clean-tmp
-	-@ $(foreach dir,$(TMP),docker cp $(IMAGE):/opt/system/$(dir) $(dir) 2>/dev/null;)
-
-clean-tmp: ## Removes temporary files
-	-@ $(foreach dir,$(TMP),rm -rf $(dir);)
+assets: ## Create assets volumes
+assets: yarn
+assets: CMD = rake assets:precompile
+yarn: ## Install JS dependencies
+yarn: CMD = yarn install
```

Comment:
I know Yarn is the standard of Rails 5 (I think from Rails 5.1), but this won't work in our current setup. We need to migrate back to Yarn our Webpack entire suite.

Reply:
Why won't it work?